unit_tests/web_tests/web_test_page_objects.py

```diff
@@ -1114,6 +1119,17 @@ def image_exists(self, image):
         except TimeoutException:
             return False
 
+    # Added helper function of checkboxes operation--------------------------------
+    def is_checkbox_selected(self, checkbox_id):
```

Comment:
This method exists again above in a different class as "checkbox_selected." It should be moved to the super-class (Page) so there is only a single definition and can be used by both sub-classes.